tests/phpunit/agentic-commerce/class-wc-stripe-agentic-commerce-inventory-tracker-test.php

```
@@ -0,0 +1,535 @@
+		// Short-circuit the Files API upload.
+		add_filter(
+			'wc_stripe_agentic_commerce_files_api_pre_request',
+			function () {
+				return [ 'id' => 'file_test_123' ];
+			}
+		);
+
+		// Short-circuit the ImportSet creation.
+		add_filter(
+			'pre_http_request',
+			function ( $pre, $args, $url ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$args'. (undefined)

(UnusedFormalParameter)